airflow/config_templates/config.yml

```diff
@@ -1536,6 +1536,13 @@ webserver:
       type: boolean
       example: ~
       default: "False"
+    caching_hash_method:
+      description: |
+        The caching algorithm used by the webserver.
```

Comment:
Should add a note that this value must be something available in `hashlib`.

Reply:
Done 👍